plugins/catalog/src/hooks/useStarredEntites.ts

```diff
@@ -37,7 +37,20 @@ export const useStarredEntities = () => {
     }
   }, [observedItems?.newValue]);
 
+  const toggleStarredEntity = useCallback(
+    (entity: string) => {
```

Comment:
Following previous comments, `entity` here should probably have either different name or type